flink-libraries/flink-table/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -1869,13 +1869,23 @@ private RexNode convertOver(Blackboard bb, SqlNode node) {
 			// Walk over the tree and apply 'over' to all agg functions. This is
 			// necessary because the returned expression is not necessarily a call
 			// to an agg function. For example, AVG(x) becomes SUM(x) / COUNT(x).
+
```

Comment:
I think we can remove this file for now and rely on Calcite 1.13 implementation. We don't need it anymore.